sound/soc/sof/topology.c

```diff
@@ -2307,9 +2308,8 @@ static int sof_widget_ready(struct snd_soc_component *scomp, int index,
 
 	swidget->core = comp.core;
 
-	ret = sof_parse_tokens(scomp, &swidget->comp_ext, comp_ext_tokens,
-			       ARRAY_SIZE(comp_ext_tokens), tw->priv.array,
-			       le32_to_cpu(tw->priv.size));
+	ret = sof_parse_tokens(scomp, swidget, comp_ext_tokens, ARRAY_SIZE(comp_ext_tokens),
+			       tw->priv.array, le32_to_cpu(tw->priv.size));
```

Comment:
This change should with
```patch
-               offsetof(struct sof_ipc_comp_ext, uuid), 0, NULL},
+               offsetof(struct snd_sof_widget, uuid), 0, NULL},
```
I know you did it in another commit, but it should belong to this commit IMHO.

Reply:
good catch. I originally had it in this commit. this is the result of a bad rebase. fixed now